modules/recommended-configurable-storage-technology.adoc

```diff
@@ -84,39 +84,47 @@ In a non-scaled/high-availability (HA) {product-title} registry cluster deployme
 
 * The preferred storage technology is object storage followed by block storage. The
 storage technology does not have to support RWX access mode.
-* The storage technology must ensure read-after-write consistency. All NAS storage are not
-recommended for {product-title} Registry cluster deployment with production workloads.
-* While `hostPath` volumes are configurable for a non-scaled/HA {product-title} Registry, they are not recommended for cluster deployment.
+* The storage technology must ensure read-after-write consistency. File storage is
+not recommended for {product-title} Registry cluster deployment with production
```

Comment:
For consistency, perhaps `Registry` here should be lower-cased? The other instances of `{product-title} registry cluster deployment` in this doc are all lower-cased.